go.mod

```diff
@@ -53,3 +53,5 @@ require (
 replace github.com/alecthomas/chroma => github.com/wakatime/chroma v0.10.0-wakatime.1
 
 replace github.com/matishsiao/goInfo => github.com/wakatime/goInfo v0.1.0-wakatime.8
+
+replace github.com/kevinburke/ssh_config => github.com/wakatime/ssh_config v1.2.0-wakatime.1
```

Comment:
Why do we need to replace this library?

Reply:
We need this new public function recently added so we can mock the ssh config file location:
https://github.com/kevinburke/ssh_config/commit/5724bd127982289dd52e79933bbe60b98a5b639e#diff-0e426a43248661127a0c0ee115aef7a1093b635f8993b3f7ebb1dd9f05b8f249R267

Reply:
Updated to use master instead of fork.